docs/core/preview3/tools/dotnet-run.md

```diff
@@ -24,30 +24,28 @@ dotnet-run -- Runs source code 'in-place' without any explicit compile or launch
 
 ## Synopsis
 
-`dotnet run [--help] [--framework] [--configuration]
-    [--project] [[--] [application arguments]]`
+```
+dotnet run <project>
+dotnet run [-f|--framework] [-c|--configuration]
+dotnet run [[--] [application arguments]]
+dotnet run [-h|--help]
+```
 
 ## Description
-The `dotnet run` command provides a convenient option to run your application from the source code with one command. 
-It compiles source code, generates an output program and then runs that program. 
-This command is useful for fast iterative development and can also be used to run a source-distributed program (for example, a website).
+The `dotnet run` command provides a convenient option to run your application from the source code with one command. It is useful for fast iterative development in the command line. The command depends on [`dotnet build`](dotnet-build.md) command to build the code, so any requirements for the build, such as that the project has to be restored first, apply to `dotnet run` as well. 
 
-This command relies on [dotnet build](dotnet-build.md) to build source inputs to a .NET assembly, before launching the program. 
-The requirements for this command and the handling of source inputs are all inherited from the build command. 
-The documentation for the build command provides more information on those requirements.
+Output files are written out into the default location which is `bin/<configuration>/<target>`. For example, if you have a `netcoreapp1.0` application and you run `dotnet run`, the output will be placed in `bin/Debug/netcoreapp1.0`. Files are owverwritten as needed. Temporary files are placed in the `obj` directory. 
 
-Output files are written to the child *bin* folder, which will be created if it doesn't exist. 
-Files will be overwritten as needed. 
-Temporary files are written to the child *obj* folder.  
-
-In case of a project with multiple specified frameworks, `dotnet run` will first select the .NET Core frameworks. If those do not exist, it will error out. To specify other frameworks, use the `--framework` argument.
+In case of a project with multiple specified frameworks, `dotnet run` will show an error unless `--framework` option is used to specify for which framework to run the application.
```

Comment:
In case of a project with multiple specified frameworks -> When a project specifies multiple frameworks,